src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -147,14 +147,12 @@ extension CachedAppConfiguration: AppConfigurationProviding {
 			Log.debug("fetching cached app configuration", log: .appConfig)
 			// use the cached version
 			return Just(cachedVersion)
-				.receive(on: DispatchQueue.main)
 				.eraseToAnyPublisher()
 		} else {
 			Log.debug("fetching fresh app configuration. forceFetch: \(forceFetch), force: \(force)", log: .appConfig)
 			// fetch a new one
 			return getAppConfig(with: store.appConfigMetadata?.lastAppConfigETag)
 				.map({ $0.config })
-				.receive(on: DispatchQueue.main)
```

Comment:
Wont this make Ui-Tests fail 🤔 Just asking.

Reply:
Obviously not. 
Not working on the main thread or in the main queue does not imply automatic disaster. If his is enabled, the `supportedCountries` Publisher never finishes - try it for yourself.

Reply:
Hmm not returning on the main thread after calling the CachedAppConfiguration resulted in a UI-Test disaster in the past. What is different here? On which thread is the closure of the sink called now?

Reply:
I hope somebody wrote a test for that disaster!?

Reply:
We had a lot disaster in the past by changing code and not returning on the main thread. The solution so far, was to change back and return on main thread again. Because everything else would explode in effort.
Thats why I'm a little but concerned about this change. If it sometimes does not return on main, we might get problems somewhere.